src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -141,6 +143,11 @@ private async ValueTask InitializeAsync(PackageManagerModel model, INuGetUILogge
             _settingsKey = await GetSettingsKeyAsync(CancellationToken.None);
             UserSettings settings = LoadSettings();
             InitializeFilterList(settings);
+            var isReadmeTabEnabled = await _nugetFeatureFlagService.IsFeatureEnabledAsync(NuGetFeatureFlagConstants.RenderReadmeInPMUI);
+            _packageDetail._packageMetadataControl.InitializedReadmePreviewViewModel(Model.Context.ServiceBroker);
```

Comment:
I would recommend method names be verbs.

```suggestion
            _packageDetail._packageMetadataControl.InitializeReadmePreviewViewModel(Model.Context.ServiceBroker);
```